src/mbgl/text/glyph_set.hpp

```diff
@@ -9,12 +9,12 @@ class GlyphSet {
 public:
     void insert(uint32_t id, SDFGlyph&&);
     const std::map<uint32_t, SDFGlyph> &getSDFs() const;
-    const Shaping getShaping(const std::u16string &string, float maxWidth, float lineHeight,
+    const Shaping getShaping(const std::u16string &string, const bool rightToLeft, float maxWidth, float lineHeight,
                              float horizontalAlign, float verticalAlign, float justify,
                              float spacing, const Point<float> &translate) const;
     void lineWrap(Shaping &shaping, float lineHeight, float maxWidth, float horizontalAlign,
                   float verticalAlign, float justify, const Point<float> &translate,
-                  bool useBalancedIdeographicBreaking) const;
+                  bool useBalancedIdeographicBreaking, const bool rightToLeft) const;
```

Comment:
We've been moving towards using enums like `enum class WritingDirection : bool { LeftToRight, RightToLeft };` to avoid ambiguity in functions like this.